pallets/ethereum-xcm/src/lib.rs

```diff
@@ -142,27 +165,61 @@ pub mod pallet {
 				EthereumXcmTransaction::V1(v1_tx) =>  v1_tx.gas_limit.unique_saturated_into(),
 				EthereumXcmTransaction::V2(v2_tx) =>  v2_tx.gas_limit.unique_saturated_into()
 			}
-		}))]
+		}).saturating_add(T::DbWeight::get().reads(2)))]
```

Comment:
Why 2 reads here and 1 above?

Reply:
We divide between two costs here:

- Whatever is done at the EVM (defined by gas_limit)
- Whatever we do before entering the EVM. In the case of transact_through_proxy, we do 2 reads: ensure EthereumXcm is not suspended and ensure the proxy exists. 

In the case of transacting without proxy, we only do 1: ensure EthereumXcm is not suspended